docs/metrics.md

```diff
@@ -198,3 +198,9 @@ how long it takes to persist a chunk (and chunks preceeding it)
 this is subject to backpressure from the store when the store's queue runs full
 * `tank.total_points`:  
 the number of points currently held in the in-memory ringbuffer
+* `input.kafka-mdm.partition.%d.offset`:   
+The current offset for the partition (%d) that we have consumed.
```

Comment:
can we call this `input.kafka-mdm.partition.%d.consumer_offset` so that it's more clear for someone just looking at the metric